piety-promoted-v1/modernized/08-chapter-07.adoc

```diff
@@ -515,7 +515,7 @@ which before were in parables.
 He searched the Holy Scriptures, and found it was the good old way,
 though new to him and others;
 and the Scriptures were opened by the spirit that gave them forth,
-and a witness for God he had in himself, according to John 1:10-5. "`He that believes,
+and a witness for God he had in himself, according to John 1:5,10. "`He that believes,
```

Comment:
I had looked this up in the original and put down just as it was there, but thinking about it more, I looked it up different ways and realized this is 1 John 5:10 so I will have to change it. :)